modules/database/src/ioc/dbtemplate/msi.cpp

```diff
@@ -493,7 +499,15 @@ static void inputErrPrint(const inputData *const pinputData)
     EXIT;
 }
 
-static void inputOpenFile(inputData *pinputData, const char * const filename)
+static int isPathRelative(const char * const path) {
+#ifdef _WIN32
+    return PathIsRelativeA(path);
+#else
+    return path && path[0] != '/';
```

Comment:
Is `PathIsRelativeA(NULL)` well defined?  Or alternately, is the NULL test for the !win32 case necessary?

I had a brief look yesterday, but wasn't able to convince myself that `path` would never be NULL.

Reply:
I added a null check there as well. It can't hurt, and the documentation there isn't 100% clear to me either.